src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+type Props = {
+  user: User | null,
+};
```

Comment:
```suggestion
type Props = {
  user: User,
};
```
Don't render a `UserInfo` component at all if you don't have a user. As in your case you render an empty link with invalid values